docs/data-modeling/admin-data-model.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Data modeling for admins"
+redirect_from:
+  - /docs/latest/administration-guide/03-metadata-editing
+---
+
+# Data modeling for admins
```

Comment:
Thoughts on alternative titles for this page? 

I think "metadata editing" might be too abstract (compared to the name of the page in the product, as well as the task of managing metadata itself). 

The Admin Data Model feels more closely related to business logic (semantic data models) than traditional "data modelling" (performant data models). 

You need metadata for both types of modelling (like data types), but the use cases are very different. I've tried to make that clearer in this doc, but I think the title and descriptions could still use some work (as well as the connective bits in other docs).